scripts/003-gdb-PPU.sh

```diff
@@ -1,19 +1,21 @@
 #!/bin/sh -e
+#
 # gdb-PPU.sh by Naomi Peori (naomi@peori.ca)
+# Modified by luizfernandonb (luizfernando.nb@outlook.com)
 
-GDB="gdb-7.5.1"
+GDB="gdb-10.1"
```

Comment:
Did you test gdb 10.1 that it works?  Or only that it compiles?

Reply:
I couldn't test it, I tried to follow the steps in [this tutorial](https://sourceware.org/gdb/current/onlinedocs/gdb/Connecting.html#:~:text=In%20target%20remote%20mode%2C%20if,GDB%20connected%20to%20the%20target.) and I couldn't because I don't know the port of my PS3, if you can help me how to test the GDB, I'll be grateful, but I'm 90% sure it will work because of this [here](https://stackoverflow.com/questions/29801755/c-do-i-need-gcc-and-gdb-with-the-same-version-to-debug)

Reply:
Do you know what is the PS3 port for GDB to connect to?